doc/user-guide.adoc

```diff
@@ -912,6 +917,107 @@ data:
 type: kubernetes.io/tls
 ----
 
+=== Affinity
+
+==== Node Affinity
+
+===== Basic node affinity
+
+Using `nodeAffinityLabels` field it is possible to set required labels for pod scheduling on specific nodes:
+
+[source,yaml]
+----
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: my-app
+  namespace: test
+spec:
+  applicationImage: quay.io/my-repo/my-app:1.0
+  affinity:
+    nodeAffinityLabels:
+      customNodeLabel: label1, label2
+      customNodeLabel2: label3
+----
+
+===== Advanced node affinity example:
+
+Require a large node type and prefer zone A and zone B
+
+[source,yaml]
+----
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: my-app
+  namespace: test
+spec:
+  applicationImage: quay.io/my-repo/my-app:1.0
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key:  node.kubernetes.io/instance-type
+            operator: In
+            values:
+            - large
+      preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 60
+        preference:
+          matchExpressions:
+          - key: failure-domain.beta.kubernetes.io/zone
+            operator: In
+            values:
+            - zoneA
+      - weight: 20
+        preference:
+          matchExpressions:
+          - key: failure-domain.beta.kubernetes.io/zone
+            operator: In
+            values:
+            - zoneB
+----
+
+==== Pod Affinity and Anti-Affinity
+
+Pod affinity and anti-affinity allow you to constrain which nodes your pod is eligible to be scheduled based on labels on pods that are already running on the node rather than based on labels on node.
```

Comment:
Change to "Use pod affinity and anti-affinity to constrain which nodes your pod can be scheduled on. Add labels to pods that are already running on the node instead of putting labels on the node.".